app_server/app/util/Utility.js

```diff
@@ -60,3 +61,62 @@ exports.createLogger = function (filename) {
     ]
   });
 };
+
+var clearUserProfile =
```

Comment:
In the future we might have lots of nested objects, for example seeing subscribers' streams or something and so I thought it would be cleaner to keep all of these formatting functions inside Utility.